apis/k8ssandra/v1alpha1/cassandraconfig_types.go

```diff
@@ -51,14 +51,14 @@ type JvmOptions struct {
 	// Cass Config Builder: supported for Cassandra 4.0 in jvm-server.options.
 	// Corresponds to: -Xms.
 	// +optional
-	InitialHeapSize *resource.Quantity `json:"heap_initial_size,omitempty" cass-config:"^3.11.x:jvm-options/initial_heap_size;>=4.x,dse@>=6.8.x:jvm-server-options/initial_heap_size"`
+	InitialHeapSize *resource.Quantity `json:"heap_initial_size,omitempty" cass-config:"^3.11.x:jvm-options/initial_heap_size;>=4.x,dse@>=6.8.x,hcd@>=1.x.x:jvm-server-options/initial_heap_size"`
```

Comment:
Question: I haven't gone through this line by line, I assume you've just done a regex find and replace and ensured that anything which is HCD related uses `jvm-server-options` in the same way that DSE does?